src/main/java/frc/robot/Calibrations.java

```diff
@@ -37,4 +36,12 @@ public static final class SwerveModuleCalibrations {
   public static final double INTAKE_INTAKING_POWER = 1.0;
   public static final double INTAKE_EJECTION_POWER = -1.0;
   public static final double AUTO_CLAMP_WAIT_TIME_SECONDS = 0.5;
+
+  /** This should be [0,1] */
+  public static final double MAX_CHARGE_STATION_CLIMB_SPEED = 0.5 / Constants.SwerveDrive.MAX_SPEED_METERS_PER_SECOND;
+  /** Conversion factor between the robot pitch (in degrees) to a velocity (in [0,1]) for balancing on the charge station */
+  public static final double CHARGE_STATION_PITCH_DEGREES_TO_VELOCITY = Constants.MAX_PITCH_DEGREES / MAX_CHARGE_STATION_CLIMB_SPEED;
```

Comment:
also needs units. MPS is okay for meters per second since that's so long. or meters_per_sec

Reply:
Isn't this one in [0,1]?

Reply:
Now I'm confused; the variables here and in AutoChargeStationBalance are named w/ mps but are used like [-1,1]